generic-templates/newsletter-signup-snippet/snippet.html

```diff
@@ -1,1491 +1,1516 @@
-      </div>
-      <div class="row">
-        <select class="country-input" required="required">
-          <option value="" selected="selected">
-            {{ countryPlaceholder }}
-          </option>
-
-          <option value="US">
-            United States
-          </option>
-
-          <option value="AF">
-            Afghanistan
-          </option>
-
-          <option value="AL">
-            Albania
-          </option>
-
-          <option value="DZ">
-            Algeria
-          </option>
-
-          <option value="AS">
-            American Samoa
-          </option>
-
-          <option value="AD">
-            Andorra
-          </option>
-
-          <option value="AO">
-            Angola
-          </option>
-
-          <option value="AI">
-            Anguilla
-          </option>
-
-          <option value="AG">
-            Antigua and Barbuda
-          </option>
-
-          <option value="AR">
-            Argentina
-          </option>
-
-          <option value="AM">
-            Armenia
-          </option>
-
-          <option value="AW">
-            Aruba
-          </option>
-
-          <option value="AU">
-            Australia
-          </option>
-
-          <option value="AT">
-            Austria
-          </option>
-
-          <option value="AZ">
-            Azerbaijan
-          </option>
-
-          <option value="BS">
-            Bahamas
-          </option>
-
-          <option value="BH">
-            Bahrain
-          </option>
-
-          <option value="BD">
-            Bangladesh
-          </option>
-
-          <option value="BB">
-            Barbados
-          </option>
-
-          <option value="BY">
-            Belarus
-          </option>
-
-          <option value="BE">
-            Belgium
-          </option>
-
-          <option value="BZ">
-            Belize
-          </option>
-
-          <option value="BJ">
-            Benin
-          </option>
+      <div class="transition-container">
+        <div class="transition-track">
+          <div class="transition-item">
+            <input class="email-input form-input" type="email" required="required" placeholder="{{ email_placeholder }}"></input>
+            <button class="next-button"></button>
+          </div>
+          <div class="transition-item">
+            <button class="back-button"></button>
+            <input class="first-name-input form-input" type="text" required="required" placeholder="{{ firstname_placeholder }}"></input>
+            <button class="next-button"></button>
+          </div>
+          <div class="transition-item">
+            <button class="back-button"></button>
+            <select class="country-input form-input" value="US" required="required">
+              <option value="" selected="selected">
+                {{ country_placeholder }}
+              </option>
+
+              <option value="US">
+                United States
```

Comment:
Moving US into it's alphabetic spot is enough? Is that what you meant by properly sorted? If so, I'm fine with that seeing I also don't live in the US ;)